libs/store_helper/ebpf_store_helper.c

```diff
@@ -11,25 +11,34 @@
 #include "ebpf_store_helper.h"
 #include "ebpf_windows.h"
 
-uint32_t
+#ifndef USER_MODE
+#define _EBPF_RESULT(x) (NT_SUCCESS(x) ? EBPF_SUCCESS : EBPF_FAILED)
+#else
+#define _EBPF_RESULT(x) win32_error_code_to_ebpf_result(x)
+#endif
```

Comment:
Can this perhaps go in libs\platform\{user,kernel}\framework.h ?
(Then maybe we wouldn't the USER_MODE define)

Reply:
Including it in `framework.h` would take care of KM, for for UM? What options shall we go for, `ebpf_registry_helper_um.h` or `ebpf_utilities.h` my suggestion)?

Reply:
Actually, I'd disagree in spreading these macros out, as how they work are *specific* to the store's needs, not necessarily for other current/future uses. So actually, I'd just keep them in `ebpf_registry_helper_km/um.c`.

Reply:
I disagree they are specific to the store's needs.  If you leave them this way, there's a 90% chance I'll move them as part of issue #2677